### PR TITLE
Improve victory anim in The Pit slightly

### PR DESCRIPTION
**Pull request by toberge**

### CODE REVIEW

**Comment by toberge** [comment]
couldn't resist changing things in Crater Town as well >:)

File: Assets/Scenes/CraterTown.unity [hunk 1]
```diff
@@ -38,7 +38,7 @@ RenderSettings:
   m_ReflectionIntensity: 1
   m_CustomReflection: {fileID: 0}
   m_Sun: {fileID: 662445327}
-  m_IndirectSpecularColor: {r: 0.06254172, g: 0.21942095, b: 0.49505305, a: 1}
+  m_IndirectSpecularColor: {r: 0.062285002, g: 0.21922584, b: 0.4949425, a: 1}
```
Comment (Fueredoriku) [inline Assets/Scenes/CraterTown.unity]: Unity moment 1

File: Assets/Scenes/ThePit.unity [hunk 2]
```diff
@@ -38,7 +38,7 @@ RenderSettings:
   m_ReflectionIntensity: 1
   m_CustomReflection: {fileID: 0}
   m_Sun: {fileID: 1421388808}
-  m_IndirectSpecularColor: {r: 0.056538448, g: 0.21520689, b: 0.49377772, a: 1}
+  m_IndirectSpecularColor: {r: 0.06380783, g: 0.22031887, b: 0.4952789, a: 1}
```
Comment (Fueredoriku) [inline Assets/Scenes/ThePit.unity]: Unity moment 2

**Review by Fueredoriku — APPROVED** [review]
Looks nice! 🤠 

You could probably revert the Unity moment 1 and 2 in the comments (since they are the only changes to those files) if you have the energy, but other than that this is great 👍

**Pull request merged**